users/users-model.js

```diff
@@ -139,4 +139,4 @@ function edituserProfile(Userid, body){
   .update(body, "id")
 }
 
-myUndefinedFunction();
\ No newline at end of file
+// myUndefinedFunction();
```

Comment:
We can delete this since it's an artifact from testing the Sentry exception tool.